databricks/koalas/frame.py

```diff
@@ -491,19 +493,20 @@ def _reduce_for_stat_function(self, sfun, name, axis=None, numeric_only=False):
                         # Pass in both the column and its data type if sfun accepts two args
                         col_sdf = sfun(col_sdf, col_type)
                     exprs.append(col_sdf.alias(name_like_string(label)))
+                    new_column_labels.append(label)
 
             sdf = self._sdf.select(*exprs)
-            pdf = sdf.toPandas()
-
-            if self._internal.column_labels_level > 1:
-                pdf.columns = pd.MultiIndex.from_tuples(self._internal.column_labels)
 
-            assert len(pdf) == 1, (sdf, pdf)
+            # The data is expected to be small so it's fine to transpose/use default index.
```

Comment:
How about adding a column like `F.lit(0).alias(SPARK_DEFAULT_INDEX_NAME)` and `index_map=[(SPARK_DEFAULT_INDEX_NAME, None)]` without default index?